roles/common.nix

```diff
@@ -14,8 +14,6 @@
   environment.systemPackages = [
     # for quick activity overview
     pkgs.htop
-    # for users with TERM=xterm-termite
-    pkgs.termite.terminfo
```

Comment:
cc @zimbatm I had to remove it for now since termite is broken in nixpkgs-unstable.